client/quantum_serverless/core/job.py

```diff
@@ -723,3 +723,20 @@ def save_result(result: Dict[str, Any]):
         logging.warning("Something went wrong: %s", response.text)
 
     return response.ok
+
+
+def _map_status_to_serverless(status: str) -> str:
+    """Map a status string from job client to the Qiskit terminology."""
+    status_map = {
+        "PENDING": "INITIALIZING",
+        "RUNNING": "RUNNING",
+        "STOPPED": "CANCELED",
+        "SUCCEEDED": "DONE",
+        "FAILED": "ERROR",
+        "QUEUED": "QUEUED",
+    }
+
+    try:
+        return status_map[status]
+    except KeyError as exc:
+        raise KeyError(f"Cannot interpret unknown job status: {status}") from exc
```

Comment:
Maybe we shouldn't error, maybe we should just print the status like you suggested? Not really sure

Reply:
I think printing is less invasive :)